src/components/contextpopup/partials/contextpopup.html

```diff
@@ -28,9 +28,9 @@
               <td>MGRS</td>
               <td>{{coordmgrs}}</td>
           </tr>
-          <tr ng-hide="!altitude">
+          <tr ng-if="altitude">
               <td translate>elevation</td>
-              <td>{{altitude}} [m]</td>
+              <td>{{altitude | measure:'distance':[' m']:1}}</td>
```

Comment:
Maybe not. In PR/Issue referenced above it was wished to round to meter. Now to dm again. The question remains.